### PR TITLE
Greek Translations

### PR DESCRIPTION
**Pull request by billpcs**
I created the Greek Translations to Scala and Racket. There may be small mistakes which I hope that will be pointed out and fixed as soon as possible

### CODE REVIEW

File: el-gr/racket-gr.html.markdown [hunk 1]
```diff
@@ -0,0 +1,745 @@
+---σ
```
Comment (vendethiel) [inline el-gr/racket-gr.html.markdown]: extra char here

File: el-gr/racket-gr.html.markdown [hunk 2]
```diff
@@ -0,0 +1,745 @@
...
+  - ["th3rac25", "https://github.com/voila"]
+  - ["Eli Barzilay", "https://github.com/elibarzilay"]
+  - ["Gustavo Schmidt", "https://github.com/gustavoschmidt"]
+  - ["Duong H. Nguyen", "https://github.com/cmpitg"]
+  - ["Keyan Zhang", "https://github.com/keyanzhang"]
+translators:
+    - ["Vasilis Panagiotopoulos" , "https://github.com/billpcs/"]
+---
+
+Racket is a general purpose, multi-paradigm programming language in the Lisp/Scheme family.
```
Comment (vendethiel) [inline el-gr/racket-gr.html.markdown]: You should probably translate this

File: el-gr/racket-gr.html.markdown [hunk 3]
```diff
@@ -0,0 +1,745 @@
...
+
+Feedback is appreciated! You can reach me at [@th3rac25](http://twitter.com/th3rac25) or th3rac25 [at] [google's email service]
```
Comment (vendethiel) [inline el-gr/racket-gr.html.markdown]: You should probably remove this

Comment (billpcs) [inline el-gr/racket-gr.html.markdown]: Done

**Comment by vendethiel** [comment]
Thanks!

**Comment by vendethiel** [comment]
:heart_eyes: :heart_eyes_cat:

**Comment by billpcs** [comment]
Thanks for the merge :)

**Pull request merged**